docs-v2/integrations/all/shopify.mdx

```diff
@@ -40,6 +40,7 @@ _No setup guide yet._
 -   [OAuth-related docs](https://shopify.dev/docs/apps/auth/oauth)
 -   [List of OAuth scopes](https://shopify.dev/docs/api/usage/access-scopes#authenticated-access-scopes)
 -   [API](https://shopify.dev/docs/api/admin)
+-   [SCIM API Docs](https://help.shopify.com/en/manual/organization-settings/users/security/scim)
```

Comment:
Last thing that I missed from before. At the bottom of this file should be a reference to the connect docs as seen here https://github.com/NangoHQ/nango/blob/master/docs-v2/integrations/all/algolia.mdx#going-further


It should say specifically Shopify SCIM similar to https://github.com/NangoHQ/nango/blob/master/docs-v2/integrations/all/trello.mdx